x-pack/plugins/beats_management/common/constants/beats_table.ts

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const TABLE_CONFIG = {
+  INITIAL_ROW_SIZE: 5,
+  PAGE_SIZE_OPTIONS: [3, 5, 10, 20],
+  ACTIONS: {
+    BULK_ASSIGN_TAG: 'BULK_ASSIGN_TAG',
+    BULK_DELETE: 'BULK_DELETE',
+    BULK_EDIT: 'BULK_EDIT',
```

Comment:
no longer needed

Reply:
heh the BULK_EDIT is no longer needed. comment thing won't let me say that exactly

Reply:
Yeah.. so I removed this in https://github.com/elastic/kibana/pull/21182/commits/f4ed8bc336a80c53d70ebbbae7187df10ef86eba but it's still showing in the diff, dunno why.